Makefile

```diff
@@ -1,30 +1,23 @@
 PACKAGE_NAME?=github.com/projectcalico/node
-GO_BUILD_VER?=v0.51
-
-# These need to be defined before the common makefile is included due to the way
-# the common makefile expands certain variables. (If they are not pre-defined then
-# the variables won't expand correctly in pre-requisites.)
-BUILD_IMAGE?=calico/node
-RELEASE_IMAGES?=gcr.io/projectcalico-org/node eu.gcr.io/projectcalico-org/node asia.gcr.io/projectcalico-org/node us.gcr.io/projectcalico-org/node
-PUSH_IMAGES?=$(BUILD_IMAGE) quay.io/calico/node
+GO_BUILD_VER?=v0.53
 
 ORGANIZATION=projectcalico
 SEMAPHORE_PROJECT_ID?=$(SEMAPHORE_NODE_PROJECT_ID)
 
-###############################################################################
-# Download and include Makefile.common
-#   Additions to EXTRA_DOCKER_ARGS need to happen before the include since
-#   that variable is evaluated when we declare DOCKER_RUN and siblings.
-###############################################################################
-MAKE_BRANCH?=$(GO_BUILD_VER)
-MAKE_REPO?=https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)
+RELEASE_REGISTRIES    ?=gcr.io/projectcalico-org eu.gcr.io/projectcalico-org asia.gcr.io/projectcalico-org us.gcr.io/projectcalico-org
+RELEASE_BRANCH_PREFIX ?=release
+DEV_TAG_SUFFIX        ?=0.dev
 
-Makefile.common: Makefile.common.$(MAKE_BRANCH)
-	cp "$<" "$@"
-Makefile.common.$(MAKE_BRANCH):
-	# Clean up any files downloaded from other branches so they don't accumulate.
-	rm -f Makefile.common.*
-	curl --fail $(MAKE_REPO)/Makefile.common -o "$@"
+# If this is a release, also tag and push additional images.
+ifeq ($(RELEASE),true)
+NODE_IMAGE     ?=calico/node
```

Comment:
does this need to be just `node` since the `gcr.io/projectcalico-org` registries store the image at `gcr.io/projectcalico-org/node`?

Reply:
Thanks for catching that!